### DISCUSSION TITLE
Reorder files in pull request UI

### DISCUSSION THREAD

**Discussion by nealsid** [community/community#31268] [Pull Requests]
I have a feature request that may or may not be more widely useful, but thought I'd throw it out there.  In this pull request commit:

https://github.com/opcm/pcm/pull/434/commits/fc0f7e44ab0948f81703230c0b0ca6d7a935c921

A removal of a structure is shown below updated references to a new replacement structure.  I thought it could ease code review if the file with the structure removal diff was at the top, and if I could manually place the files in a particular order, say, via, drag and drop, it would be easier to understand the changes referring to the new structure, rather than scrolling down to the bottom and then back up.  Thank you.

**Comment by brahn** [comment]
Another use case:

In a project with automated tests, it is increasingly common for those files to appear in the same directory as the source code they test rather than a standalone tests directory.  E.g.:

```
my-project
├── src
│   ├── module1
│   │   ├── module1.ts
│   │   ├── module1.test.ts
│   ├── module2
│   │   ├── module2.ts
│   │   ├── module2.test.ts
```

However, the default ordering of files in a pull request results in this code appearing in a less-than-legible order:

```
src/module1/module1.test.ts
src/module1/module1.ts
src/module1/module2.test.ts
src/module1/module2.ts
```

It would be helpful if the pull request author could reorder these files to present them to the reviewer in the most approachable manner.
